gkeys/gkeys/actions.py

```diff
@@ -311,7 +333,6 @@ def installkey(self, args):
                 self.gpg.set_keydir(gkey.keydir, "recv-keys")
                 self.gpg.set_keyseedfile()
                 seeds = self.gpg.seedfile.seeds
-                #print(seeds)
                 if seeds:
                     self.logger.debug("ACTIONS: installkey; found installed seeds:"
                         "\n %s" % seeds)
```

Comment:
Many of the seed actions still needed updating.  So please do not add more print()s.  

Please use the output variable (self.output()) for normal info being output.  In fact convert any seed actions you touch to using self.output() instead of the print() that were present before you starting making changes.